src/backends/torch/native/templates/nbeats.h

```diff
@@ -364,8 +364,15 @@ namespace dd
           else if ((pos = s.find(_season_str)) != std::string::npos)
             {
               _stack_types.push_back(seasonality);
-              _thetas_dims.push_back(
-                  std::stoi(s.substr(pos + _season_str.size())));
+              int val = -1;
+              try
+                {
+                  val = std::stoi(s.substr(pos + _season_str.size()));
+                }
+              catch (std::invalid_argument &e)
+                {
```

Comment:
Why error is ignored here? can we still put a number after "s" e.g "s512" ? if so, I think we should be reading this number only if the string is not "s", and keep the error

Reply:
ok, should be bettre now

Reply:
![image](https://user-images.githubusercontent.com/3530657/110947259-39744980-8340-11eb-907e-df417ce3d547.png)


The only usage of the word I could find !

Reply:
https://en.wiktionary.org/wiki/bettre